src/renderer/Generics/redesign/Header/Header.tsx

```diff
@@ -109,14 +124,33 @@ export const Header = (props: NodeOverviewProps) => {
             )}
           </div>
         )}
-        <Button
-          {...startStopButtonProps}
-          variant="icon-left"
-          size="small"
-          type={
-            startStopButtonProps.iconId === 'play' ? 'primary' : 'secondary'
-          }
-        />
+        {/* In case of the status being an error, we should show both start & stop buttons so the user can try
```

Comment:
should we confirm this behavior with Danny? I don't think we have this case in the design

Reply:
@ddannehh :) This "error" case should be very rare and hopefully we can remove this in the future.

Reply:
Is it a kind of error where we don't know what the actual problem is? Having a start/stop button at the same time could be somewhat confusing. Is it a possibility to have a "↻ Retry" button that does the necessary procedure in the background?

Reply:
Is it a kind of error where we don't know what the actual problem is? Having a start/stop button at the same time could be somewhat confusing. Is it a possibility to have a "↻ Retry" button that does the necessary procedure in the background?